src/components/alert/index.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames'
+
+import { IconInfo16 } from '@hashicorp/flight-icons/svg-react/info-16'
+import { IconAlertTriangle16 } from '@hashicorp/flight-icons/svg-react/alert-triangle-16'
+import { IconAlertDiamond16 } from '@hashicorp/flight-icons/svg-react/alert-diamond-16'
+
+import { AlertProps, AlertData } from './types'
+import s from './alert.module.css'
+
+const ALERT_DATA: AlertData = {
+	tip: { title: 'Tip', icon: <IconInfo16 /> },
+	highlight: { title: 'Tip', icon: <IconInfo16 /> },
+	note: { title: 'Note', icon: <IconAlertTriangle16 /> },
+	warning: { title: 'Warning', icon: <IconAlertDiamond16 /> },
+}
+
+export default function Alert({ children, type = 'tip', title }: AlertProps) {
+	const data = ALERT_DATA[type]
+
+	if (!children) {
+		throw new Error(
+			'[Alert]: No `children` found, please pass a description body'
+		)
+	}
+
+	if (!data) {
```

Comment:
Since this is a 'fresh' component, I thought it would be a good opportunity to actually throw an error for content authors. but open to feedback here. the danger is that authors don't check their work and we get a clientside error?

Reply:
I like guard rails!